src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -69,6 +69,7 @@
     <None Remove="Views\GuidedTour\HtmlPages\Resources\ConnectTheNode.gif" />
   </ItemGroup>
   <ItemGroup>
+  <PackageReference Include="AvalonEdit" Version="4.3.1.9430" CopyXML="true"/>
```

Comment:
why was this needed?

Reply:
this packageRefecence was moved from DynamoUtilities. DynamoCoreWpf.csproj is the project that actually uses it.
I think it was used by DynamoUtilities only for an indirect reference to `System.Collections.Immutable`.
I refactored it so that we do not need it in DynamoUtilities.

This might cause compile issues...if someone was using `AvalonEdit` APIs by referencing the DynamoUtilities project ...But from what I was able to check...looks like we are ok with the autodesk integrators.